daemon/container.go

```diff
@@ -34,13 +34,13 @@ import (
 )
 
 var (
-	ErrNotATTY                 = errors.New("The PTY is not a file")
-	ErrNoTTY                   = errors.New("No PTY found")
-	ErrContainerStart          = errors.New("The container failed to start. Unknown error")
-	ErrContainerStartTimeout   = errors.New("The container failed to start due to timed out.")
+	// ErrContainerRootfsReadonly is returned when a container
+	// rootfs is marked readonly. (perhaps move to archive.go and
+	// make private, the only site of usage)
```

Comment:
Does that mean you agree on moving it?

For a rename, ErrRootFSReadOnly? I cannot think of a way to make that much shorter, and I do not think "Container" adds anything informative. Errors like this can also be replaced simply with fmt.errorf as well, because they have no internal variables, only a non-specific message.